### PR TITLE
Generator fixes

### PR DESCRIPTION
**Pull request by kylebuch8**
- isPfelement logic change
- gulp factory argument fix
- adding a few properties to package.json
- version bump

### CODE REVIEW

**Review by castastrophe — APPROVED** [review]
Code looks good, just need to check out the branch and run it locally

**Comment by castastrophe** [comment]
@kylebuch8 After running this locally, I'm seeing some formatting and output issues.  Here is the output issue in the package.json file:
```
"version": "1.0.0-prerelease.19",
  "description": " element for PatternFly Elements",
  "keywords": [
    "web-components",
    "html"
  ],
  "repository": {
    "type": "git",
    
    "url": ""
    
  },
```

Note the line breaks being wonky, the url not picking up, and the name of the element being left out of the description.

**Review by castastrophe — CHANGES_REQUESTED** [review]
Seeing some output issues when running it locally

File: generators/templates/package.json [hunk 1]
```diff
@@ -6,9 +6,16 @@
   },
   "version": "<%= pfelementVersion %>",
   "description": "<%= readmeName %> element for PatternFly Elements",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
   "repository": {
     "type": "git",
+    <% if (isPfelement) { %>"url": "github:patternfly/patternfly-elements",
```
Comment (castastrophe) [inline generators/templates/package.json]: Be sure to use a line-slurp here, the spacing is coming out wonky in the output file.  The readmeName doesn't seem to be working either.

Comment (kylebuch8) [reply generators/templates/package.json]: I think I fixed this in my latest commit.

Comment (kylebuch8) [reply generators/templates/package.json]: I didn't see any issues with the README though. I have tested this both in and out of the PatternFly Elements repository without any issues.

Comment (castastrophe) [reply generators/templates/package.json]: Twas my poor naming of the element - closing this out and approving.

**Review by castastrophe — APPROVED** [review]
👍 Let Go, Telling @mwcz

**Pull request merged**